app-frontend/src/app/pages/projects/edit/annotate/annotate.scss

```diff
@@ -79,19 +79,37 @@
 }
 
 .annotate-clone-marker {
-    background: rgba(255, 68, 51, 0.5);
+    background: rgba(229, 115, 115, 0.5);
     border-radius: 50%;
-    border: 2px solid rgba(	255, 21, 0, 0.8);
+    border: 2px solid rgba(229, 115, 115, 0.8);
 }
 
 .annotate-highlight-marker {
-    background: rgba(255, 255, 255, 0);
+    background: rgba(229, 115, 115, 0);
     border-radius: 50%;
-    border: 2px solid rgba(	255, 21, 0, 0.8);
+    border: 2px solid rgba(229, 115, 115, 0.8);
 }
 
 .annotate-hover-marker {
     background: rgba(115, 143, 252, 0.8);
     border-radius: 50%;
-    border: 2px solid rgba(	255, 21, 0, 1);
+    border: 2px solid rgba(229, 115, 115, 1);
+}
+
+rf-annotate-sidebar-item{
+    &.is-sidebar-item-hovered{
+        .list-group-item.list-sidebar-drawn-annotations{
+            border: 2px solid $brand-primary;
+        }
+    }
+    &.is-sidebar-item-not-hovered{
```

Comment:
it's kind of weird to have both a negative and positive class - usually you have one as the default style, then override the relevant parts when a condition is met:
ex
```css
.base-class {
  some-style: some-val;
  &.active {
   some-style: other-val;
 }
}
```